### PR TITLE
Fix: return the owner of a table's username instead of their id

### PR DESCRIPTION
**Pull request by TAnas0**
## What is the Purpose?
Fix of the Dots Frontend issue: https://github.com/hikaya-io/dots-frontend/issues/162

## What was the approach?
Return the owner of a table as a username instead of an ID

## Are there any concerns to addressed further before or after merging this PR?
If this endpoint is used  somewhere else (`tables/table_uuid`), this change needs to be accounted for.

## Mentions?
@andrewtpham @nasirhjafri @ninetteadhikari

### CODE REVIEW

File: tables/serializers.py [hunk 1]
```diff
@@ -29,5 +32,8 @@ def get_data(self, obj):
             del data['_id']
         return data
 
-
-
+    def get_owner(self, obj):
+        """
+        Replace the `owner` attribute with the username of the user
+        """
+        return obj.owner.username
```
Comment (nasirhjafri) [inline tables/serializers.py:39]: Please use prefetch on the query set.

File: tables/serializers.py [hunk 2]
```diff
@@ -3,6 +3,8 @@
 from .models import Table
 from .utils import connect_to_mongo
 
+from tables.models import Table
```
Comment (nasirhjafri) [inline tables/serializers.py:6]: Duplicate import.

File: tables/serializers.py (hunk 1 above)
Comment (TAnas0) [reply tables/serializers.py:39]: @nasirhjafri I am not sure what you mean or how to implement it. Could you please take care of it if it is not much of a hassle for you?

Comment (andrewtpham) [reply tables/serializers.py:37]: @nasirhjafri push a commit to this PR to solve this issue

**Pull request merged**